test/e2e/subscription_e2e_test.go

```diff
@@ -3390,7 +3389,11 @@ func updateInternalCatalog(t GinkgoTInterface, c operatorclient.ClientInterface,
 			fmt.Println("catalog updated")
 			return true
 		}
-		fmt.Printf("waiting for catalog pod %v to be available (after catalog update) - %s\n", catalog.GetName(), catalog.Status.GRPCConnectionState.LastObservedState)
+		msg := fmt.Sprintf("waiting for catalog pod %v to be available (after catalog update) - %s\n", catalog.GetName(), catalog.Status.GRPCConnectionState.LastObservedState)
+		if msg != logMsg {
```

Comment:
global nit: the other funcs expose time stamps to help communicate the passage of time